fix: change enable_credit_and_industry_pathways default to true (#2644)
edx/edx-enterprise: CHANGELOG.rst

----------
* nothing unreleased

[8.1.1] - 2026-06-18
---------------------

> * fix: enable Credit and Industry Pathways by default for enterprise customers

Reviewer: Use the commit message
```suggestion
* fix: change enable_credit_and_industry_pathways default to true
```

Author: resolved
